moveit_setup_assistant/include/moveit/setup_assistant/tools/compute_default_collisions.h

```diff
@@ -40,7 +40,7 @@
 #include <moveit/macros/class_forward.h>
 namespace planning_scene
 {
-MOVEIT_CLASS_FORWARD(PlanningScene)  // Defines PlanningScenePtr, ConstPtr, WeakPtr... etc
+MOVEIT_CLASS_FORWARD(PlanningScene);  // Defines PlanningScenePtr, ConstPtr, WeakPtr... etc
```

Comment:
Why do you need to forward declare this as opposed to just including the appropriate header from moveit_planning?  I don't think there is a circular header relationship with this header from moveit_planning.